helm/charts/config/templates/configmaps.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata: 
+  name: {{ include "config.fullname" . }}-config-cm
+  namespace: {{ .Values.global.namespace }}
+data:
+  ADMIN_PW:  {{ .Values.adminPass }}
+  EMAIL: {{ .Values.email }}
+  DOMAIN: {{ .Values.domain }}
+  ORG_NAME: {{ .Values.orgName }}
+  COUNTRY_CODE: {{ .Values.countryCode }}
+  STATE: {{ .Values.state }}
+  CITY: {{ .Values.city }}
+  LDAP_TYPE: {{ .Values.ldapType }}
```

Comment:
Is this suppose to represent GLUU_PERSISTENCE_TYPE env ? If so the env needs to be changed here to `GLUU_PERSISTENCE_TYPE `